src/BizHawk.Client.Common/movie/tasproj/ZwinderStateManagerSettings.cs

```diff
@@ -109,6 +111,19 @@ public ZwinderStateManagerSettings(ZwinderStateManagerSettings settings)
 		[Description("Where to keep the reserved states.")]
 		public IRewindSettings.BackingStoreType AncientStoreType { get; set; } = IRewindSettings.BackingStoreType.Memory;
 
+		public enum States

```

Comment:
If you're serializing this as `byte`, it should be denoted in the enum definition: `public enum States : byte`.